docs/howto/odk-add-orcidio-module.md

```diff
@@ -0,0 +1,100 @@
+## Adding an ORCIDIO import to your ontology with ODK
+
+The [Open Researcher and Contributor Identifier (ORCID)](https://orcid.org/) is a global, unambiguous way to identify a researcher.
+ORCID URIs (e.g., https://orcid.org/0000-0003-4423-4370) can therefore be used to unambigously and actionably attribute various aspects of ontology terms in combination with DC Terms or IAO predicates. However, URIs themselves are opaque and it is difficult to disambiguate to which person an ORCID corresponds when browsing an ontology (e.g., in Protégé).
+
+[ORCIDIO](https://github.com/cthoyt/orcidio) is an ontology that declares ORCID URIs as named individuals and associates basic metadata (e.g., name, description) to each such that tools like Protégé can display a human-readable label rather than the URI itself as in the following example.
+
+![](https://github.com/cthoyt/orcidio/raw/main/img/screenshot-2.png)
+
+In this guide, we discuss how to add ORCIDIO to your ODK setup.
+
+#### 1. Include ORCIDIO as an import into the ODK config file
+
+In your ODK configuration (e.g. `src/ontology/myont-odk.yaml`), add the following to the `import_group`:
+
+```yaml
+import_group:
+  annotation_properties:
+    - rdfs:label
+    - dc:description
+    - dc:source
+    - IAO:0000115
+  products:
+    - id: orcidio
+      mirror_from: https://w3id.org/orcidio/orcidio.owl
+      module_type: filter
+      base_iris:
+        - https://orcid.org/
+```
+
+The list of annotation properties, in particular `dc:source`, is important for the `filter` module to work (ORCIDIO relies heavily on axiom annotations for provenance).
+
+#### 2. Update your catalog
+
+TODO: "as usual" should be re-written to cross-link to another guide about updating the catalog (or don't say as usual to keep this more self-contained)
```

Comment:
TODO: "as usual" should be re-written to cross-link to another guide about updating the catalog (or don't say as usual to keep this more self-contained)